zerver/openapi/zulip.yaml

```
@@ -3151,6 +3163,10 @@ paths:
                                 **Changes**: Deprecated in Zulip 4.0 (feature level 54), replaced by
                                 the `realm_linkifiers` event type, which has a clearer name and format,
                                 instead.
+
+                                **Changes**: In Zulip 7.0 (feature level 175), support for
+                                linkifiers in the legacy format was dropped by unconditionally
+                                returning an empty list at all times.
```

Newest change first:

```suggestion
                                **Changes**: In Zulip 7.0 (feature level 175), support for
                                linkifiers in the legacy format was dropped by unconditionally
                                returning an empty list at all times.

                                Deprecated in Zulip 4.0 (feature level 54), replaced by
                                the `realm_linkifiers` event type, which has a clearer name and format,
                                instead.
```

That's important so that when the reader sees a version number that's older than they care about, they can stop reading.

---

Also, this is a spot where the main description needs updating so that it reflects the current state. Which is that this event is never sent at all, right?

---

That's right. This has been updated.